### DISCUSSION TITLE
210x210mm Bed

### DISCUSSION THREAD

**Discussion by bonz099** [xpDIY/3DPlacer#2] [General]
Since Its hard to print edge to edge on my printer with 220x220 print bed, I made a slightly smaller version that's 210x210. 
[3dplacer - Modified bed-210x210mm.zip](https://github.com/xpDIY/3DPlacer/files/11593602/3dplacer.-.Modified.bed-210x210mm.zip)

**Comment by xpDIY** [comment]
Thanks for sharing the version, this make it possible for 220mm printer to print, would be interesting to know how that work out for you, if the print out is strong enough, and the tolerance, can the other part fits well, etc.

Would you be ok that I put the file into the repo? that way other people can also test out, or even better, you could create a pull request, that tracks your contribution better.

**Comment by bonz099** [comment]
Thank you for the merge!
